index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+	<meta name="viewport" content="width=device-width">
+</head>    
+<body>
+    <nav class="container">
+		<div class="navContainer">
+			<div class="banner">
+				<h1>Luke Chaston</h1>
+				<h2>Web Developer | Programmer </h2>
+			</div>
+			<section class="buttonContainer">
+				<div class="button"><a href="index.html#biography">About Me</a></div>
```

Comment:
the links don't appear to be working - try  href="#biography" to see if that works instead